packages/caip/src/adapters/mtpelerin/index.ts

```diff
@@ -123,6 +122,7 @@ const chainIdToMtPelerinNetworkCodeMap: Record<ChainId, string> = {
   [ethChainId]: 'mainnet',
   [btcChainId]: 'bitcoin_mainnet',
   [avalancheChainId]: 'avalanche_mainnet', // this is actually the C-Chain
+  [optimismChainId]: 'optimism_mainnet',
```

Comment:
Confirmed she's the one 
<img width="477" alt="image" src="https://user-images.githubusercontent.com/17035424/212957447-e73bb02e-8fae-4c01-9bba-2a8acd8074e6.png">